erpnext/accounts/doctype/pos_invoice/pos_invoice.py

```diff
@@ -377,7 +380,7 @@ def set_pos_fields(self, for_validate=False):
 				)
 				customer_group_price_list = frappe.db.get_value("Customer Group", customer_group, 'default_price_list')
 				selling_price_list = customer_price_list or customer_group_price_list or profile.get('selling_price_list')
-				if customer_currency != profile.get('currency'):
+				if customer_currency and customer_currency != profile.get('currency') and not self.currency:
```

Comment:
```suggestion
				if customer_currency and customer_currency != profile.get('currency'):
```

Customer Currency should overwrite existing currency in the Invoice, right?
Is this intentional? If yes, Can you help me understand with an example?